platform/vs/docker-sonic-vs/start.sh

```diff
@@ -44,13 +48,44 @@ mkdir -p /etc/swss/config.d/
 
 rm -f /var/run/rsyslogd.pid
 
-supervisorctl start rsyslogd
+supervisord_cfg="/etc/supervisor/conf.d/supervisord.conf"
+chassis_cfg_file="/usr/share/sonic/virtual_chassis/default_config.json"
+chassis_cfg_file_default="/etc/default/sonic-db/default_chassis_cfg.json"
+host_template="/usr/share/sonic/templates/hostname.j2"
+db_cfg_file="/var/run/redis/sonic-db/database_config.json"
+db_cfg_file_tmp="/var/run/redis/sonic-db/database_config.json.tmp"
+
+if [ -r "$chassis_cfg_file" ]; then
+   echo $(sonic-cfggen -j $chassis_cfg_file -t $host_template) >> /etc/hosts
+else
+   chassis_cfg_file="$chassis_cfg_file_default"
+   echo "10.8.1.200 redis_chassis.server" >> /etc/hosts
+fi
 
 mkdir -p /var/run/redis/sonic-db
 cp /etc/default/sonic-db/database_config.json /var/run/redis/sonic-db/
 
+supervisorctl reread
```

Comment:
Is this needed? We just need to restart the redis-chassis service.

Reply:
verified we don't need the change, it is a legacy before we switch to bind with redis-chassis server hostname instead of ip address.